### PR TITLE
Réinitialisation du mdp bénéficiaire - Page choix de la méthode

### PR DESCRIPTION
**Pull request by AntoineAresu**
[Ticket](https://trello.com/c/NcGPUQLS/3721-5-pro-r%C3%A9initialisation-du-mdp-b%C3%A9n%C3%A9ficiaire-page-choix-de-la-m%C3%A9thode)

image
image

### CODE REVIEW

**Review by tcheymol — APPROVED** [review]

File: src/ControllerV2/BeneficiaryResetPassword.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#[IsGranted('ROLE_MEMBRE')]
+class BeneficiaryResetPassword extends AbstractController
+{
+    #[IsGranted('UPDATE', 'beneficiary')]
+    #[Route(path: '/beneficiary/{id}/reset-password', name: 'reset_password_beneficiary', methods: ['GET'])]
+    public function choice(Beneficiaire $beneficiary): Response
+    {
+        return $this->render('v2/reset_password/beneficiary/choice.html.twig', [
+            'beneficiary' => $beneficiary,
+        ]);
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply src/ControllerV2/BeneficiaryResetPassword.php:19]: #[Template()] attribute have been introduced in Symfony 6.2

File: src/ControllerV2/BeneficiaryResetPassword.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace App\ControllerV2;
+
+use App\Entity\Beneficiaire;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[IsGranted('ROLE_MEMBRE')]
+class BeneficiaryResetPassword extends AbstractController
```
[the remark this answers is not in the document]
Comment (AntoineAresu) [reply src/ControllerV2/BeneficiaryResetPassword.php:11]: indeed

**Review by NilsRec — APPROVED** [review]

File: src/ControllerV2/BeneficiaryResetPassword.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (tcheymol) [reply src/ControllerV2/BeneficiaryResetPassword.php:19]: 😀 So upgrade to 6.2 :)

[the remark this answers is not in the document]
Comment (AntoineAresu) [reply src/ControllerV2/BeneficiaryResetPassword.php:19]: i'll do it in next PR

**Pull request merged**